src/System.Windows.Forms/src/System/Windows/Forms/TaskDialogButton.cs

```diff
@@ -184,6 +184,9 @@ internal TaskDialogButton(TaskDialogResult standardButtonResult)
         ///   This property can be set while the dialog is shown.
         /// </para>
         /// </remarks>
+        /// <exception cref="InvalidOperationException">
+        ///   This button is currently bound to a task dialog, but its <see cref="Visible"/> property value is <see langword="false"/>.
+        /// </exception>
```

Comment:
In response to https://github.com/dotnet/winforms/pull/1133#discussion_r407452226

I suppose we could word it as follows:
```suggestion
        /// <exception cref="InvalidOperationException">
        ///   An attempt to modify the property value when this button is currently bound to a task dialog, but its <see cref="Visible"/> property value is <see langword="false"/>.
        /// </exception>
```

@gewarren do you have a better suggestion?

Reply:
```suggestion
        /// <exception cref="InvalidOperationException">
        ///  An attempt is made to set this value on a button that is bound to a task dialog whose <see cref="Visible"/> property value is <see langword="false"/>.
        /// </exception>
```

Also, does the exception also occur if you're setting Enabled to false?

Reply:
Thanks, I think I will use this phrase.

Note: I found two other phrases in the existing documentation for property setters:
[`ReadOnlyCollection<T>.IList<T>.Item[Int32]`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.readonlycollection-1.system-collections-generic-ilist-t--item?view=netcore-3.1): Always thrown *if the property is set*.
[`SortedList.Item[Object]`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.sortedlist.item?view=netcore-3.1): *The property is set* and the SortedList object is read-only. - or - *The property is set,* key does not exist in the collection, and the SortedList has a fixed size.


> Also, does the exception also occur if you're setting Enabled to false?

Yes, it occurs on any value in the described circumstance.

Reply:
Nice find! We can use the [`SortedList.Item[Object]`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.sortedlist.item?view=netcore-3.1) wording variant too.

Reply:
For the `Item` properties, you could also consider wording like:

Always thrown if an attempt is made to set the value.
An attempt is made to set the value and the `SortedList` object is read-only.
An attempt is made to set the value, the key does not exist in the collection, and the `SortedList` has a fixed size.

Reply:
Thank you! I chose the phrase with "The property is set and ..." as it was easier for me to express.